tutorials/18_diagnostics_lc2st.ipynb

```diff
@@ -0,0 +1,821 @@
+      "text/plain": [
+       "<Figure size 1200x300 with 3 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# PP-Plots\n",
+    "from sbi.analysis.plot import pp_plot_lc2st\n",
+    "\n",
+    "fig, axes = plt.subplots(1,len(thetas_star), figsize=(12,3))\n",
+    "for i in range(len(thetas_star)):\n",
+    "    probs_data, _ = lc2st.get_scores_on_observed_data(\n",
+    "        theta_o=post_samples_star[i],\n",
+    "        x_o=xs_star[i],\n",
+    "        return_probs=True\n",
+    "    )\n",
+    "    probs_null, _ = lc2st.get_statistics_under_null_hypothesis(\n",
+    "        theta_o=post_samples_star[i],\n",
+    "        x_o=xs_star[i],\n",
+    "        return_probs=True\n",
+    "    )\n",
+    "\n",
+    "    pp_plot_lc2st(\n",
+    "        probs=[probs_data],\n",
+    "        probs_null=probs_null,\n",
+    "        conf_alpha=conf_alpha,\n",
+    "        labels=[\"Classifier probabilities \\n on observed data\"],\n",
+    "        colors=[\"red\"],\n",
+    "        ax=axes[i],\n",
+    "    )\n",
+    "    axes[i].set_title(f\"PP-plot for observation {i+1}\")\n",
+    "axes[i].legend()\n",
```

Comment:
Note, the confidence region does not contain the true null. In the paper's figure 4, this is the case. It might cause some confusion that the dashed line in these plots is not contained in the grey CI area.

Reply:
You'e right, this is because we might be too discriminative when using ensembling. I will change the number of ensemble models such that the dashed lines are in the gret CR area.

Reply:
However, I wonder if this is necessary? While I agree that this can be confusing, the theoretical null is not what we compare to in practice.. 

I'm just curious, I agree to keep the larger CR including the true null for the tutorial.